pkg/sql/exec/coldata/vec.go

```diff
@@ -29,6 +29,9 @@ type column interface{}
 type Vec interface {
 	Nulls
 
+	// Type returns the type of datums stored in this Vec.
```

Comment:
nit: we haven't used `datums` yet in this code - could you just say data, to avoid eventual terminology clash with tree.Datum?